### PR TITLE
Add rusbase extension

### PR DESCRIPTION
**Pull request by khlebobul**
## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/rusbase/package.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  "icon": "extension-icon.png",
+  "author": "khlebobul",
+  "categories": [
+    "News"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Sections",
```
Comment (pernielsentikaer) [inline extensions/rusbase/package.json:15]: Can we add something like Search/View/List before?

File: extensions/rusbase/package.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      "description": "Sections News from Rusbase",
+      "mode": "view"
+    },
+    {
+      "name": "markets",
+      "title": "Markets",
```
Comment (pernielsentikaer) [inline extensions/rusbase/package.json:21]: Can we add something like Search/View/List before?

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

- At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by khlebobul** [comment]
@pernielsentikaer Good evening, it's just a media resource in a different language. You mean I need to translate the searchBar items into English?

**Comment by pernielsentikaer** [comment]
Yes, that's not allowed according to the [documentation](https://developers.raycast.com/basics/prepare-an-extension-for-store#localization-language)

**Comment by khlebobul** [comment]
Okay, I'll fix it soon, thank you

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Comment by khlebobul** [comment]
Hello, for some reason there is another file package-lock.json outside the extension folder. I can't delete it. Could you please help me with this problem to get the extension into the store?

File: extensions/rusbase/src/markets.tsx [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+    },
+  });
+
+  return (
+    <List
+      isLoading={isLoading}
+      searchBarAccessory={
+        <List.Dropdown
+          tooltip="Select a category"
+          defaultValue={"Беспилотные аппараты"}
```
Comment (pernielsentikaer) [inline extensions/rusbase/src/markets.tsx:41]: Not sure what this is, can you correct that too?

**Review by pernielsentikaer — APPROVED** [review]

**Pull request merged**